lib/tbot/service_spiffe_svid_output.go

```diff
@@ -134,8 +197,35 @@ func (s *SPIFFESVIDOutputService) generate(ctx context.Context) error {
 		s.botCfg.CertificateTTL,
 	)
 	if err != nil {
+		return nil, nil, trace.Wrap(err)
+	}
+	return res, privateKey, nil
+}
+
+func (s *SPIFFESVIDOutputService) render(
```

Comment:
What exactly does the word `render` mean in this context? Does it mean "write to the config"? "Write to the certificate store"? Please make it more explicit and/or add a doc comment.

Reply:
It's a bit of a standard within this package, all of the Output services will have some code that fetches all the necessary data, and then they call the `render` to actually convert that into some config files and write them. Previously the `SPIFFESVIDOutputService` was a little odd in that it just had a `generate` and no `render`.